docker-compose.yml

```diff
@@ -169,6 +169,7 @@ services:
     image: compliance-backend-rails
     entrypoint: ''
     command: 'bundle exec rake ssg:import_rhel_supported'
+    restart: on-failure
```

Comment:
@skateman here's how I 'solved' it. Ideally, we'd do a real check (like `Host.count` even) and wait to execute the rake task until it succeeds, but this also ensures the SSGs are imported by leveraging docker-compose, even if it fails the first time.

Reply:
Isn't this overcomplicated? Wouldn't be just using a single container that first sets up the DB and then imports the SSGs solve the problem? Or is there a specific reason we these things as separate containers? For me it feels like they are the part of the same DB setup :thinking:

Reply:
There's data setup and schema setup, and they're separate in our deployments. This docker-compose is meant to approximate our deployments; of course, the cyndi setup doesn't ever need to happen in any other environments other than devel and smoke, so in that way devel is a bit more complicated than normal.